include/spirv-tools/optimizer.hpp

```diff
@@ -57,35 +59,35 @@ class Optimizer {
   // The constructed instance will have an empty message consumer, which just
   // ignores all messages from the library. Use SetMessageConsumer() to supply
   // one if messages are of concern.
-  explicit Optimizer(spv_target_env env);
+  SPIRV_TOOLS_OPT_EXPORT explicit Optimizer(spv_target_env env);
```

Comment:
You can make the whole class SPIRV_TOOLS_OPT_EXPORT instead of every function individually.

Reply:
Yes, that was my recollection.  (I haven't had to do this in some time...)
That is much preferred: mark an entire class rather than individual methods.